packages/mbed-greentea/README.md

```diff
@@ -4,8 +4,6 @@ The development of Greentea has been moved into the [mbed-os-tools](../../src/mb
 
 -------------
 
-[![Circle CI](https://circleci.com/gh/ARMmbed/greentea.svg?style=svg)](https://circleci.com/gh/ARMmbed/greentea)
-[![Coverage Status](https://coveralls.io/repos/github/ARMmbed/greentea/badge.svg?branch=master)](https://coveralls.io/github/ARMmbed/greentea?branch=master)
```

Comment:
A little too far here?

Reply:
We should still have coveralls.

Reply:
That's for the greentea package readme. We're still using coveralls, this just removes the badge for the old greentea repository.